src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -100,15 +109,27 @@ public bool uninstall(ChocolateyConfiguration configuration, PackageResult packa
             return run_action(configuration, packageResult, CommandNameType.uninstall);
         }
 
+        public void before_modify_noop(PackageResult packageResult)
+        {
+            noop_action(packageResult, CommandNameType.upgrade);
+        }
+
+        public bool before_modify(ChocolateyConfiguration configuration, PackageResult packageResult)
+        {
+            return run_action(configuration, packageResult, CommandNameType.upgrade);
+        }
+
         private string get_helpers_folder()
         {
             return _fileSystem.combine_paths(ApplicationParameters.InstallLocation, "helpers");
         }
 
         public string wrap_script_with_module(string script, ChocolateyConfiguration config)
         {
-            var installerModule = _fileSystem.combine_paths(get_helpers_folder(), "chocolateyInstaller.psm1");
-            var scriptRunner = _fileSystem.combine_paths(get_helpers_folder(), "chocolateyScriptRunner.ps1");
+            var installerModules = _fileSystem.get_files(ApplicationParameters.InstallLocation, "chocolateyInstaller.psm1", SearchOption.AllDirectories);
+            var installerModule = installerModules.FirstOrDefault();
+            var scriptRunners = _fileSystem.get_files(ApplicationParameters.InstallLocation, "chocolateyScriptRunner.ps1", SearchOption.AllDirectories);
+            var scriptRunner = scriptRunners.FirstOrDefault();

```

Comment:
The same here. I had a security fix that was removed on rebase. I fixed it prior to merge.